app/controllers/datasets.js

```diff
@@ -76,10 +76,13 @@ export default Ember.Controller.extend({
     }
   }),
   selected_rows: Ember.computed('model.years_available.@each.selected', function() {
+    if (Ember.compare(Ember.Error, this.get('model.years_available')) === 0) {
+      return this.get('model.years_available');
+    }
+    let year_column = 'row.' + this.get('model.dataset.yearcolumn');
     let years_available = this.get('model.years_available').filterBy('selected', true).map(selected => selected.year);
-    let emberModelObject = this.get('model');
     return this.get('model.raw_data.rows').filter(function(row) {
-      return years_available.includes(row.acs_year);
+      return years_available.includes(eval(year_column));
```

Comment:
The coding police have been called. `eval` to lookup a value? `eval` that is not sandboxed? There is something fundamentally odd going on here.